package.json

```diff
@@ -38,8 +38,9 @@
 		"gettext-parser": "^4.0.3",
 		"husky": "^3.0.2",
 		"identity-obj-proxy": "^3.0.0",
-		"jest": "^24.9.0",
+		"jest": "26.6.3",
```

Comment:
The explicit version was set to mirror [Gutenberg's approach](https://github.com/WordPress/gutenberg/blob/78b235a583f3ab67f82301eff635e1ff95266db5/package.json#L178), but I am unaware of a reason to avoid `^` in this circumstance. I will add the `^` to the version range.